### PR TITLE
Move Files from Unsubmitted to Submitted Directories

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift [hunk 1]
```diff
@@ -221,6 +221,17 @@ public class LocalStorage {
         }
     }
 
+    static func moveToSubmittedJobs(jobId: String) throws {
+        try createDirectory(at: submittedDirectory, overwrite: false)
+        let sourceURLs = try fileManager.contentsOfDirectory(atPath: jobId).map {
```
Comment (vanshg) [inline Sources/SmileID/Classes/Helpers/LocalStorage.swift:226]: Shouldn't atPath be unsubmitted/jobId, not just jobId?

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift [hunk 2]
```diff
@@ -221,6 +221,17 @@ public class LocalStorage {
...
+            URL(fileURLWithPath: jobId).appendingPathComponent($0)
+        }
+        for sourceURL in sourceURLs {
```
Comment (vanshg) [inline Sources/SmileID/Classes/Helpers/LocalStorage.swift:229]: instead of moving each file over individually, we can just move the entire jobId directory

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift [hunk 3]
```diff
@@ -240,7 +251,7 @@ public class LocalStorage {
         try Zip.quickZipFiles(urls, fileName: "upload")
     }
 
-    static func delete(at url: URL) throws {
+    private static func delete(at url: URL) throws {
```
Comment (vanshg) [inline Sources/SmileID/Classes/Helpers/LocalStorage.swift:254]: 👍

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift (hunk 2 above)
Comment (jumaallan) [reply Sources/SmileID/Classes/Helpers/LocalStorage.swift:229]: actually this was marked as draft because I had a different approach that I had not pushed

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift [hunk 4]
```diff
@@ -221,6 +227,13 @@ public class LocalStorage {
         }
     }
 
+    static func moveToSubmittedJobs(jobId: String) throws {
+        try createDirectory(at: submittedDirectory, overwrite: false)
+        let unsubmittedFileDirectory = try unsubmittedDirectory.appendingPathComponent(jobId)
+        let submittedFileDirectory = try submittedDirectory.appendingPathComponent(jobId)
```
Comment (vanshg) [inline Sources/SmileID/Classes/Helpers/LocalStorage.swift:233]: a little pedantic, but for slightly more accurate naming
```suggestion
        let unsubmittedJobDirectory = try unsubmittedDirectory.appendingPathComponent(jobId)
        let submittedJobDirectory = try submittedDirectory.appendingPathComponent(jobId)
```

File: Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift [hunk 5]
```diff
@@ -329,10 +329,23 @@ public class SelfieViewModel: ObservableObject, ARKitSmileDelegate {
...
+                    try LocalStorage.moveToSubmittedJobs(jobId: self.jobId)
+                } catch {
+                    print("Error moving job to submitted directory: \(error)")
+                    self.error = error
+                }
                 DispatchQueue.main.async { self.processingState = .success }
             } catch {
+                if SmileID.allowOfflineMode {
+
+                } else {
```
Comment (vanshg) [inline Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift:342]: can you remove the empty branch

Comment (jumaallan) [reply Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift:342]: I moved this logic to `LocalStorage` actually, forgot to clean this up

File: Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift [hunk 6]
```diff
@@ -217,6 +218,12 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
                 },
                 receiveValue: { response in
                     self.jobStatusResponse = response
+                    do {
+                        try LocalStorage.moveToSubmittedJobs(jobId: self.jobId)
+                    } catch {
+                        print("Error moving job to submitted directory: \(error)")
+                        self.onError(error: SmileIDError.unknown("Failed to create file"))
```
Comment (vanshg) [inline Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift:225]: We could just pass back `error` here?
